dbms/src/Columns/ColumnString.h

```diff
@@ -278,6 +299,17 @@ class ColumnString final : public COWPtrHelper<IColumn, ColumnString>
         offsets.push_back(offsets.empty() ? 1 : (offsets.back() + 1));
     }
 
+    void insertManyDefaults(size_t length) override
+    {
+        chars.resize_fill(chars.size() + length);
+        offsets.reserve(offsets.size() + length);
+        for (size_t i = 0; i < length; ++i)
+        {
+            chars.push_back(0);
```

Comment:
L304 already use `resize_fill`, why need to push `0` here again?

Reply:
It's a mistake, fix it and update the gtest to cover this.